src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/patches/RadicleToolWindow.java

```diff
@@ -21,9 +21,14 @@
 import java.awt.BorderLayout;
 
 public class RadicleToolWindow extends VcsToolWindowFactory {
-
     protected ToolWindowManagerListener toolWindowManagerListener;
     protected PatchTabController patchTabController;
+
+    @Override
+    public void init(@NotNull ToolWindow window) {
```

Comment:
Let's go through this change together please and make sure this is the correct approach.